python/hail/methods/statgen.py

```diff
@@ -2059,8 +2059,7 @@ def balding_nichols_model(n_populations, n_samples, n_variants, n_partitions=Non
     >>> bn_ds = hl.balding_nichols_model(4, 40, 150, 3,
     ...          pop_dist=[0.1, 0.2, 0.3, 0.4],
     ...          fst=[.02, .06, .04, .12],
-    ...          af_dist=TruncatedBetaDist(a=0.01, b=2.0, min=0.05, max=1.0),
-    ...          seed=1)
+    ...          af_dist=hl.rand_beta(a=0.01, b=2.0, min=0.05, max=1.0))
```

Comment:
how to seed?

Reply:
ugh, somehow some changes from unrelated commits snuck in here. should be gone now.